### PR TITLE
blog: add call-to-action blog for february CFPs

### PR DESCRIPTION
**Pull request by GregSutcliffe**
DevConf closes on Friday, so a quick turnaround would be great :)

### CODE REVIEW

File: _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+be upon us. FOSDEM! CfgMgmtCamp! DevConf! Busy, busy, busy, and we want you to
+be a part of it! Come tell us your story.
+
+For these events to be successful, we need lots of good Foreman talks - and we
+want to hear your Foreman story. These conferences aren't just for developers
+to show off the latest cool feature (although that's always nice), but rather
+serve as an opportunity for the community to share it's knowledge, swap
+stories, and learn from each other. To do that, we need a mix of talks, from
+dev previews (as mentioned) to success stories, lessons from the front line,
+plugins used to solve a problem and so on.
```
Comment (ohadlevy) [inline _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md]: I would also mention to meet up with developers?

File: _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Fosdem - both close 18th Nov
+CfgMgmt Room: https://lists.fosdem.org/pipermail/fosdem/2016-October/002459.html &
+Virt/IaaS Room: http://www.ovirt.org/blog/2016/10/call-for-proposal-fosdem-2017/
+
+DevConf - closes *11th Nov* (tomorrow!)
+https://devconf.cz/#cfp
+
+Even if you're not in the mood to present, we hope to see you there. Swing by
+the booth and say hi!
```
Comment (ohadlevy) [inline _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md]: do you happen to have any pictures from past events?

Comment (GregSutcliffe) [reply _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md]: Can't find much, only https://twitter.com/iNecas/status/695562152019628032. Do you have any from the Drug Opera meal?

File: _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md (hunk 1 above)
Comment (GregSutcliffe) [reply _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md]: Updated to mention meeting face-to-face and to raise the emphasis on non-dev submissions.

File: _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md (hunk 2 above)
Comment (GregSutcliffe) [reply _posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md]: Done, and added some emphasis on key parts of the message

**Comment by ohadlevy** [comment]
thanks @GregSutcliffe

**Pull request merged**